src/components/AgentAuthority/IssueLicense/IssueLicense.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Polls for verified presentation proofs and issues a license, then deletes the proof
+ */
+import { useEffect, useState } from 'react'
+import moment from 'moment'
+import LicenseCredDef from '../LicenseCredDef'
+import useGetLoopedPresentProofRecords from '../../../interface/hooks/use-get-looped-present-proof-records'
+import usePostIssueCredentialSendOffer from '../../../interface/hooks/use-post-issue-credential-send-offer'
+import useDeleteRecord from '../../../interface/hooks/use-delete-record'
+import Error from '../../Common/Misc/Error'
+
+export default function IssueLicense({ origin }) {
+  const [licenseCredDefId, setLicenseCredDefId] = useState('')
+  const [statusRecords, errorRecords, startGetRecordsHandler] =
+    useGetLoopedPresentProofRecords()
+  const [errorIssueCred, startFetchHandlerCredOffer] =
+    usePostIssueCredentialSendOffer()
+  const [errorDelete, startDeleteHandler] = useDeleteRecord()
+
+  useEffect(() => {
+    const sendCredOffers = (verifiedProofs) => {
+      verifiedProofs.forEach((verified) => {
+        const {
+          by_format: {
+            pres: {
+              indy: {
+                requested_proof: { revealed_attrs: attributes },
+              },
+            },
+          },
+        } = verified
+
+        const expiry = moment().add(2, 'years').format('YYYYMMDD')
+
+        startFetchHandlerCredOffer(
+          origin,
+          verified.connection_id,
+          licenseCredDefId,
+          attributes['0_id_uuid'].raw,
+          attributes['0_type_uuid'].raw,
+          expiry,
+          () => {},
+          () => {
+            startDeleteHandler(
+              origin,
+              verified.pres_ex_id,
+              () => {},
+              () => {}
+            )
```

Comment:
This is um ... interesting. I get it's not your fault @jonmattgray

Reply:
empty functions like this are gone now